extensions-contrib/distinctcount/src/test/java/io/druid/query/aggregation/distinctcount/DistinctCountGroupByQueryTest.java

```diff
@@ -128,20 +185,70 @@ public void testGroupByWithDistinctCountAgg() throws Exception
         query
     );
 
-    List<Row> expectedResults = Arrays.asList(
-        GroupByQueryRunnerTestHelper.createExpectedRow(
-            "1970-01-01T00:00:00.000Z",
-            client_type, "iphone",
-            "UV", 2L,
-            "rows", 2L
-        ),
-        GroupByQueryRunnerTestHelper.createExpectedRow(
-            "1970-01-01T00:00:00.000Z",
-            client_type, "android",
-            "UV", 1L,
-            "rows", 1L
-        )
-    );
+    List<Row> expectedResults;
+    if (NullHandlingHelper.useDefaultValuesForNull()) {
```

Comment:
Could it be refactored to not produce 4-time code repetition? It's a great pain to support such test code later.

Reply:
IMO, in long run we would only want to support SQL type NULL Handling, and when we make that default, this can be removed.

Reply:
Please leave a comment